src/Tools/AnalyzerRunner/AnalyzerRunner.csproj

```diff
@@ -8,9 +8,35 @@
     <!-- Automatically generate the necessary assembly binding redirects -->
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
+
+  <Choose>
```

Comment:
📝 This entire block copied from ILTools.csproj. I'm not sure why it's required, but without it `<PlatformTarget>` ends up set to x86 and AnalyzerRunner runs as a 32-bit process.

Reply:
Specifying `<PlatformTarget>AnyCPU</PlatformTarget>` should suffice.


---
In reply to: [289938504](https://github.com/dotnet/roslyn/pull/36113#discussion_r289938504) [](ancestors = 289938504)

Reply:
➡️ Submitted #36192 to simplify this